scss/forms/_form-check.scss

```diff
@@ -12,6 +12,12 @@
     float: left;
     margin-left: $form-check-padding-start * -1;
   }
+
+  // Boosted mod: Remove margin when it's the last element of a form.
+  &:last-child {
+    margin-bottom: 0;
```

Comment:
I'm not convinced by this rule, need to think about it. There's no such rule in Bootstrap and it might lead to discrepancies while using libraries based on Bootstrap (just by switching the CSS to Boosted one).